lib/omniauth-wsfed/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module WSFed
-    VERSION = '0.2.3.1'
+    VERSION = '0.2.3.2'
```

Comment:
@ad-ys How we will release the new tag for the gem?

Reply:
after merge, we just draft a release thru here https://github.com/yardstick/omniauth-wsfed/releases